### PR TITLE
Port plugin to sbt {1.0.x, 0.13.16}

### PR DESCRIPTION
**Pull request by suhasgaddam**
This requires coordination with https://github.com/47deg/sbt-org-policies/pull/510 and https://github.com/47deg/sbt-microsites/pull/221.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+  - gem install sass
+  - gem install jekyll -v 3.2.1
+
 script:
-- sbt orgScriptCI
+  - sbt ^^$SBT_VERSION orgScriptCI
+
 after_success:
-- sbt orgAfterCISuccess
+  - sbt ^^$SBT_VERSION orgAfterCISuccess
```
Comment (suhasgaddam) [inline .travis.yml]: Thinking more about the eventual `publishMicrosite`, it should only happen one of the sbt builds.
And also bumping `version.sbt`/whatever commits travis does after a merge to master.

File: project/plugins.sbt [hunk 2]
```diff
@@ -1 +1 @@
-addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.4.17")
\ No newline at end of file
+addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.6.0-SNAPSHOT")
```
Comment (fedefernandez) [inline project/plugins.sbt]: `0.6.1`?

File: project/ProjectPlugin.scala [hunk 3]
```diff
@@ -21,46 +21,47 @@ object ProjectPlugin extends AutoPlugin {
       sbtPlugin := true,
       libraryDependencies ++= Seq(
         %%("github4s"),
-        "com.47deg" %% "org-policies-core" % "0.4.17"),
-      addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
+        "com.47deg" %% "org-policies-core" % "0.6.0-SNAPSHOT"),
```
Comment (fedefernandez) [inline project/ProjectPlugin.scala]: `0.6.1`?

File: .travis.yml (hunk 1 above)
Comment (fedefernandez) [reply .travis.yml]: Yes, I agree. We could use a similar approach like the one we've used with the different Scala versions:
https://github.com/47deg/sbt-org-policies/blob/master/core/src/main/scala/sbtorgpolicies/runnable/model.scala#L25

**Review by fedefernandez — APPROVED** [review]
Looks good to me

**Comment by suhasgaddam** [comment]
@fedefernandez can you give this another check? https://github.com/47deg/sbt-dependencies/pull/30/commits/a023e1e839cd7b81038985f3ba86e5891f98761b

I need to run `orgUpdateDocFiles` and `depUpdateDependencyIssues`, but they need to be guarded so they only run for 1 sbt version.

`release` currently publishes for both sbt versions.

I also bumped to `0.2.0-SNAPSHOT` similar to how sbt-org-policies had the minor version bumped.

**Review by fedefernandez — APPROVED** [review]
New changes looks good to me :+1: thanks

**Pull request merged**